dvc/proc/manager.py

```diff
@@ -63,17 +65,62 @@ def spawn(self, args: Union[str, List[str]], name: Optional[str] = None):
             pid,
         )
 
-    def send_signal(self, name: str, signal: int):
-        """Send `signal` to the specified named process."""
-        raise NotImplementedError
-
-    def kill(self, name: str):
-        """Kill the specified named process."""
-        raise NotImplementedError
-
-    def terminate(self, name: str):
-        """Terminate the specified named process."""
-        raise NotImplementedError
+    if sys.platform == "win32":
```

Comment:
```suggestion
    if os.name == "nt":
```

In the event that we are running in cygwin, we should still use the windows specific methods (`sys.platform` would be `cygwin` and not `win32`)

Reply:
But one problem here, in the signal class, the signal constants' defination follows `sys.platform == "win32"`. So If we use `os.name == "nt"` here we would face a constants mismatch problem in `cygwin`?

Reply:
It looks like I was mistaken on the `os.name`/`sys.platform` difference, I did some testing and for cygwin `os.name == posix`, so it won't make a difference either way.